runtime/queries/godot-resource/injections.scm

```diff
@@ -11,8 +11,41 @@
   (property 
     (path) @_is_code
     (string) @injection.content))
-  (#match? @_type "type")
+  (#eq? @_type "type")
   (#match? @_is_shader "Shader")
   (#eq? @_is_code "code")
   (#set! injection.language "glsl")
 )
+
+((section 
+  (identifier) @_is_resource
+  (property 
+    (path) @_is_code
+    (string) @injection.content))
+  (#eq? @_is_resource "resource")
+  (#eq? @_is_code "code")
+  (#set! injection.language "glsl")
+)
+
+((section 
+  (identifier) @_id
+  (property 
+    (path) @_is_expression
+    (string) @injection.content))
+  (#eq? @_id "sub_resource")
+  (#eq? @_is_expression "expression")
+  (#set! injection.language "glsl")
+)
+
+((section 
+  (attribute 
+    (identifier) @_type
+    (string) @_is_shader)
+  (property 
+    (path) @_is_code
+    (string) @injection.content))
+  (#eq? @_type "type")
+  (#match? @_is_shader "GDScript")
```

Comment:
Same thing here about `#eq?`, unless `GDScript` can be a substring

Reply:
"GDScript" is a substring

In tscn file , code example:
```godot-resouce
[sub_resource type="GDScript" id="GDScript_5h5a2"]
script/source = "extends Node2D
func _ready() -> void:
	pass
func _process(delta: float) -> void:
	pass
"
```

If use `#eq` with `@_is_shader`, catch `"GDScript"`.
 `(#eq? @_is_shader "\"GDScript\"")`
 
 If need change it, I will.